### ISSUE TITLE
Icon Name's gone

### ISSUE THREAD

**Issue by P4yn33** [0xda568/uwd-oss#2]
Hy,
i'm on windows 11 pro insider.
Sadly im losing all desktop icon names when installing.
But the watermark is also gone :D

**Comment by 0xda568** [comment]
That doesn't look like a bug to me, it's officially a feature :D 

Jokes aside, thanks for opening issue, I'm gonna look into that. Did the installation / removal work smoothly without any issues?

**Comment by P4yn33** [comment]
It worked smoothly after i set the authorization to full for admins in the registry path

**Comment by Zero3K** [comment]
![image](https://github.com/0xda568/uwd-oss/assets/6194332/3e43a28d-c4cd-42a1-b199-eec1dc554514)

**Comment by Nanju4g** [comment]
Having the same problem! Is it coming from different Windows language version? (My computer is Chinese but cannot test English version because of system limitation)

**Comment by Zero3K** [comment]
No, since the issue is happening for both English and Chinese languages.

**Comment by sal-git** [comment]
For anyone having issues with Windows 11. UWD2 did the trick for me.

https://github.com/reticivis-net/uwd2
